index.d.ts

```diff
@@ -1,14 +1,56 @@
 import { Options as FocusTrapOptions } from 'focus-trap';
 import * as React from 'react';
 
+export interface FocusTrapProps extends React.AllHTMLAttributes<any> {
```

Comment:
Out of curiosity, why extend `React.AllHTMLAttributes` here? I know it was done in earlier releases as well, but as far as I can see, none of the normal HTML attributes get used for anything. For example if I use

```ts
<FocusTrap className="myFocusTrap">
  <div></div>
</FocusTrap>
```

Then the class name "myFocusTrap" doesn't get applied to any element.

Wouldn't it be better to remove this altogether?

```suggestion
export interface FocusTrapProps {
```

Reply:
It's just been like that "forever" (before my time leading this project) but now that you point this out, you're totally right: There's no point in having this extension because `<FocusTrap>` doesn't render to anything, and doesn't reflect those extra HTML props onto anything either.

And it should be safe to remove that because setting something like `className` would have never done anything anyway, so I would expect nearly no one is doing this today anyway.

Reply:
@Mathias-S PR updated!